source/wp-content/themes/wporg-developer-2023/src/style/style.scss

```diff
@@ -304,6 +304,24 @@ body[class] {
 	}
 }
 
+.wporg-reference-list,
+.wp-block-navigation .wporg-reference-list {
+	list-style: none;
+
+	> li,
+	.wp-block-navigation-item {
+		&:not(:last-child) {
+			margin-bottom: var(--wp--preset--spacing--20);
+		}
+	}
+
+	> a,
+	.wp-block-navigation-item__content {
+		color: var(--wp--custom--link--color--text);
+		text-decoration: underline;
+	}
+}
```

Comment:
Covers styles for both columns. Seemed to make more sense to keep these in sync here, rather than putting the left column styles in the block styles.